spec.html

```diff
@@ -32650,9 +32638,6 @@ <h1>Array.prototype.sort ( _comparefn_ )</h1>
           <h1>Runtime Semantics: SortCompare ( _x_, _y_ )</h1>
           <p>The SortCompare abstract operation is called with two arguments _x_ and _y_. It also has access to the _comparefn_ argument passed to the current invocation of the `sort` method. The following steps are taken:</p>
           <emu-alg>
-            1. If _x_ and _y_ are both *undefined*, return *+0*.
-            1. If _x_ is *undefined*, return 1.
-            1. If _y_ is *undefined*, return -1.
```

Comment:
`undefined`s are never part of `items`, and thus, `x` and `y` can never be `undefined`.